tests/random_test.py

```diff
@@ -223,6 +223,8 @@ def testNormalComplex(self, dtype):
     for samples in [uncompiled_samples, compiled_samples]:
       self._CheckKolmogorovSmirnovCDF(jnp.real(samples), scipy.stats.norm(scale=1/np.sqrt(2)).cdf)
       self._CheckKolmogorovSmirnovCDF(jnp.imag(samples), scipy.stats.norm(scale=1/np.sqrt(2)).cdf)
+      if config.x64_enabled:
+          self.assertTrue(samples.dtype == dtype)
```

Comment:
You may as well check the dtype in all cases, and please use `self.assertEqual(samples.dtype, dtype)`, as it results in better error messages on failure.

Reply:
done.